modules/medianetBidAdapter.js

```diff
@@ -1,8 +1,63 @@
 import { registerBidder } from 'src/adapters/bidderFactory';
 import * as utils from 'src/utils';
+import events from 'src/events';
+import CONSTANTS from 'src/constants.json';
+import { config } from 'src/config';
+import * as url from 'src/url';
 
 const BIDDER_CODE = 'medianet';
 const BID_URL = 'https://prebid.media.net/rtb/prebid';
+const EVENT_PIXEL_URL = 'lg1.media.net/log';
+const { BID_TIMEOUT } = CONSTANTS.EVENTS;
+const TIMEOUT_EVENT_NAME = 'client_timeout';
+
+let bidParams = {};
+
+events.on(BID_TIMEOUT, function (timedOutBidders) {
+  if (timedOutBidders.indexOf(BIDDER_CODE) !== -1) {
```

Comment:
we are currently not allowing bidders to listen to `BID_TIMEOUT` event.
See: https://github.com/prebid/Prebid.js/pull/2232/files#r173911280 for details.